azext_iot/constants.py

```diff
@@ -7,7 +7,7 @@
 
 import os
 
-VERSION = "0.10.7"
+VERSION = "0.10.8"
```

Comment:
same version bumping comment - check with @digimaun

Reply:
Yea its a good rule of thumb to check before committing the version in PR. For now we are good with the changes in `0.10.8`.  But keep in mind we could do an out-of-band release (due to the need of an immediate behavior change or fix) before the PR gets merged which would throw this off.